webapp/sources/rudder/rudder-web/src/main/webapp/secure/nodeManager/node.html

```diff
@@ -1,7 +1,7 @@
 <lift:surround with="common-layout" at="content">
 
 <head>
-    <title>Rudder - Search Nodes</title>
+    <title>Rudder - Node details</title>
```

Comment:
best change

Reply:
I got bored to have "search node" in my history while it was really node details. We should have added hostname, too, but it wasn't immediate